packages/paste-website/src/components/shortcodes/component-header/index.tsx

```diff
@@ -83,7 +84,7 @@ const ComponentHeader: React.FC<ComponentHeaderProps> = ({
       )}
       {shouldShowBreadcrumbs && (
         <Box marginBottom="space50">
-          {isFoundations ? <>{categoryName}</> : <Anchor href={categoryRoute}>{categoryName}</Anchor>}
+          {isFoundations || isExperiences ? <>{categoryName}</> : <Anchor href={categoryRoute}>{categoryName}</Anchor>}
```

Comment:
question: for the lower-level nested pages like `console overview` and `alarm setup`, should we be using nested breadcrumbs ("Experiences/Console" rather than just "Experiences")?